src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -43,6 +43,9 @@ class Connection
     private $consumer;
     private $autoSetup;
     private $maxEntries;
+    private $redeliveryTime = 600000;
+    private $nextClaim = 0;
+    private $claimInterval = 1000;
```

Comment:
The `redelivery_time` and `claim_interval` should be setable over query parameter of the dsn so we should move the values to `DEFAULT_OPTIONS` and set it in the construct function